src/test/java/winstone/HttpsConnectorFactoryTest.java

```diff
@@ -29,28 +30,6 @@
  * @author Kohsuke Kawaguchi
  */
 public class HttpsConnectorFactoryTest extends AbstractWinstoneTest {
-    @Test
-    public void testHttps() throws Exception {
```

Comment:
https is covered by `#wildcard()`

Reply:
Sure but `wildcard` lacks some of the assertions from `testHttps`, presumably because `testHttps` is (was, as of this PR the "primary" test for TLS and it would be pointless to test the same thing twice. But if the "primary" test for TLS is now going to be `wildcard`, which seems fine, then I think we should move those assertions to `wildcard` to retain existing coverage levels. Otherwise, we would end up with a (slight) regression in test coverage:

```java
diff --git a/src/test/java/winstone/HttpsConnectorFactoryTest.java b/src/test/java/winstone/HttpsConnectorFactoryTest.java
index 72ab567..3a613dc 100644
--- a/src/test/java/winstone/HttpsConnectorFactoryTest.java
+++ b/src/test/java/winstone/HttpsConnectorFactoryTest.java
@@ -74,7 +74,15 @@ public class HttpsConnectorFactoryTest extends AbstractWinstoneTest {
         args.put("httpsKeyStorePassword", "changeit");
         winstone = new Launcher(args);
         int port = (( ServerConnector)winstone.server.getConnectors()[0]).getLocalPort();
+        assertConnectionRefused("127.0.0.2", port);
         request(new TrustEveryoneManager(), port);
+        LowResourceMonitor lowResourceMonitor = winstone.server.getBean( LowResourceMonitor.class);
+        assertNotNull(lowResourceMonitor);
+        assertFalse(lowResourceMonitor.isLowOnResources());
+        assertTrue(lowResourceMonitor.isAcceptingInLowResources());
     }
 
     @Test
```